doc/Quality_white_wine_predictor.md

```diff
@@ -0,0 +1,245 @@
+case, a good wine quality prediction can be very useful in the
+certification phase. For example, an automatic predictive system can be
+integrated into a decision support system, helping the speed and quality
+of the performance.
+
+# Data
+
+The wine quality dataset is publicly available on the UCI machine
+learning repository (check the kinks below). The dataset has two files,
+red wine and white wine variants of the Portuguese “Vinho Verde” wine.
+It contains a large collection of datasets that have been used for the
+machine learning community. The red wine dataset contains 1599 instances
+and the white wine dataset contains 4898 instances. Both files contain
+11 input features and 1 output feature. Input features are based on the
+physicochemical tests and output variable based on sensory data is
+scaled in 11 quality classes from 0 to 10 (0-very bad to 10-very good)
+
+-   [UCI
+    repository](https://archive.ics.uci.edu/ml/datasets/Wine+Quality)
+-   [Wite wine
+    database](http://open.canada.ca/data/en/dataset/b374f60b-9580-44dc-83f6-c0a850c15f30)
+
+Input variables:
+
+1.  **Alcohol**: the amount of alcohol in wine
+2.  **Volatile acidity**: are high acetic acid in wine which leads to an
+    unpleasant vinegar taste
+3.  **Sulphates**: a wine additive that contributes to SO2 levels and
+    acts as an antimicrobial and antioxidant
+4.  **Citric Acid**: acts as a preservative to increase acidity (small
+    quantities add freshness and flavor to wines)
+5.  **Total Sulfur Dioxide**: is the amount of free + bound forms of SO2
+6.  **Density**: sweeter wines have a higher density
+7.  **Chlorides**: the amount of salt in the wine
+8.  **Fixed acidity**: are non-volatile acids that do not evaporate
+    readily
+9.  **pH**: the level of acidity
+10. **Free Sulfur Dioxide**: it prevents microbial growth and the
+    oxidation of wine
+11. **Residual sugar**: is the amount of sugar remaining after
+    fermentation stops. The key is to have a perfect balance between —
+    sweetness and sourness (wines > 45g/ltrs are sweet)
+
+# Methods
+
+**How to analyze the data** Our task here is to focus on what white wine
+features are important to get the promising result. For the purpose of
+classification model and evaluation of the relevant features, we are
+using the following algorithms to perform this task:
+
+1.  **DummyRegressor()**: is one that takes only the value 0 or 1 to
+    indicate the absence or presence of some categorical effect that may
+    be expected to shift the outcome.
+2.  **Ridge**: is a model tuning method that is used to analyse any data
+    that suffers from multicollinearity. This method performs L2
+    regularization. When the issue of multicollinearity occurs,
+    least-squares are unbiased, and variances are large, this results in
+    predicted values to be far away from the actual values.
+3.  **Random Forest**: is an ensemble learning method for
```

Comment:
Did you write these yourself? If not, the source of this information should be mentioned (even if it is just the `sklearn` documentation)